packages/@aws-cdk/aws-apigatewayv2/lib/websocket/stage.ts

```diff
@@ -13,6 +13,11 @@ export interface IWebSocketStage extends IStage {
    * The API this stage is associated to.
    */
   readonly api: IWebSocketApi;
+
+  /**
+   * The callback URL to this stage.
```

Comment:
can you describe what a callback url is, and/or link to any available docs?

Reply:
done
The most appropriate document might be https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-basic-concept.html
but some people can't understand it with just this URL, so I added the URL explaining how to use it, https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-how-to-call-websocket-api-connections.html